Partition (WBP)/Functions/VI0035H.VTI0035.fun.yml

```diff
@@ -420,40 +497,100 @@
     *
     Use Builtin(CLR_MESSAGES)
     Clr_List Named(#w_lMessages)
-    #BIFC256A := 'BEFRED'
+    #BIFC256A := 'BEFRED LT EQ NE GT'
     Add_Entry To_List(#w_lMessages)
-    #BIFC256A := 'AFTRED'
+    #BIFC256A := 'AFTRED LT EQ NE GT'
     Add_Entry To_List(#w_lMessages)
     
     #Com_Owner.mthCheck_Fetch_Trigger( #this_rc OK )
     #Com_Owner.mthIssue_Runtime_Message( #this_rc )
     
+    
     *
     * Delete
     *
     Use Builtin(CLR_MESSAGES)
     Clr_List Named(#w_lMessages)
-    #BIFC256A := 'BEFDLT'
-    Add_Entry To_List(#w_lMessages)
-    #BIFC256A := 'AFTDLT'
+    #BIFC256A := 'BEFDLT NE DATE'
     Add_Entry To_List(#w_lMessages)
     
     #Com_Owner.mthCheck_Delete_Trigger( #this_rc OK )
     #Com_Owner.mthIssue_Runtime_Message( #this_rc )
     
+    Endroutine
+    
+    
+    
+    *
+    *
+    *
+    Mthroutine Name(mthTestUnMatchTriggers)
+    Define_Map For(*BOTH) Class(#BIFA002A) Name(#this_rc)
+    
```

Comment:
Log is very plain with little info
```
 2024-02-02 00:18:57 -  Insert Trigger
    2024-02-02 00:18:57 -  - Successful
    2024-02-02 00:18:57 -  Update Trigger
    2024-02-02 00:18:57 -  - Successful
    2024-02-02 00:18:57 -  Fetch Trigger
    2024-02-02 00:18:57 -  - Successful
    2024-02-02 00:18:57 -  Delete Trigger
    2024-02-02 00:18:57 -  - Successful
```

Issue a message stating you are looking for no triggers being fired and thus no messages.

And for each check, state that 0 messages were found.

Reply:
Okay, I will improve this.

Reply:
Changes are pushed for review